backend/api/v1/v1_categories/functions.py

```diff
@@ -0,0 +1,114 @@
+import json
+from rtmis.settings import MASTER_DATA
```

Comment:
Remove MASTER_DATA. As I said, this is not like NWMIS, RTMIS will only have single source of category.json